src/Microsoft.TestPlatform.ObjectModel/Adapter/Interfaces/IFrameworkHandle2.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter
+{
+    /// <summary>
+    /// Handle to the framework which is passed to the test executors.
+    /// </summary>
+    public interface IFrameworkHandle2 : IFrameworkHandle
```

Comment:
What could be a better name? 😄

IFrameworkHandleWithAdditionalDebuggingSupport?

cc @AbhitejJohn

Reply:
As per our design review discussion, the name of this interface shall remain the same.